[gui] Make waveform amplitude values audible

gempa-jabe opened the pull request:
This PR is about to help with issue #164. It has been opened to play with various implementations and to discuss them more closely to the code rather than in issues.

gempa-jabe commented:
Where are your latest changes?

Donavin97 commented:
Hi @gempa-jabe 
I am trying to push my changes, but I think I messed up.
How do I push changes into your branch?

Donavin97 commented:
Hi @jempajabe.
I made a new pr with my changes.

gempa-jabe commented:
I have added the progress bar which sets the amplitude level in percent. Actually my screen reader only updates the progress bar every once in a while. I don't know actually when it will happen. That irregular update also happened with the version with the explicit label which I have removed. Please give it a try.

gempa-jabe commented:
Small cleanups are still necessary ...

Donavin97 commented:
Hi @gempajabe
Which cleanup would you like me to do in pickerview.cpp or pickerview.ui?
Please let me know so that I can do that ASAP.
